mlir/test/Conversion/AIRToAIE/air_channel_to_objectFifo_L1toL1.mlir

```diff
@@ -10,17 +10,17 @@
 // CHECK: module @aie.segment_0 {
 // CHECK:   %0 = AIE.tile(1, 1)
 // CHECK:   %1 = AIE.tile(1, 2)
-// CHECK:   %2 = AIE.objectFifo.createObjectFifo(%0, {%1}, 1) : !AIE.objectFifo<memref<32xi32, 2>>
```

Comment:
Is the memory space label in memref removed inside aie.segment because we assume that L1 is the only memory hierarchy that can possibly appear here?

Reply:
In this code example the aie.segment is the label of the aie.module. I can see how this is confusing, but in mlir-aie there is no segment operation like in mlir-air. 

Regarding the memory space labels, for now the objectFifo abstraction only allocates memory in L1. It can be made aware of memory in L3 through the registration of aie.external_buffers but the lowering will not allocate those itself. This is, however, subject to change with the addition of MemTiles (L2) in the future.